src/vue/forms/ChangePasswordForm.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <form
+    class="app-form change-password-form"
+    @submit.prevent="isFormValid() && showConfirmation()"
+  >
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          v-model="form.currentPassword"
+          @blur="touchField('form.currentPassword')"
+          id="change-password-current"
+          type="password"
+          :error-message="getFieldErrorMessage('form.currentPassword')"
+          :label="'change-password-form.current-password-lbl' | globalize"
+          :disabled="formMixin.isDisabled"
+        />
+      </div>
+    </div>
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          v-model="form.newPassword"
+          @blur="touchField('form.newPassword')"
+          id="change-password-new"
+          type="password"
+          :error-message="getFieldErrorMessage('form.newPassword')"
+          :label="'change-password-form.new-password-lbl' | globalize"
+          :disabled="formMixin.isDisabled"
+        />
+      </div>
+    </div>
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          v-model="form.confirmPassword"
+          @blur="touchField('form.confirmPassword')"
+          id="change-password-confirm"
+          type="password"
+          :error-message="getFieldErrorMessage('form.confirmPassword')"
+          :label="'change-password-form.confirm-password-lbl' | globalize"
+          :disabled="formMixin.isDisabled"
+        />
+      </div>
+    </div>
+    <div
+      v-if="isTotpEnabled"
+      class="app__form-row"
+    >
+      <div class="app__form-field">
+        <input-field
+          v-model="form.tfaCode"
+          @blur="touchField('form.tfaCode')"
+          id="change-password-tfa-code"
```

Comment:
why do we need an id here?

Reply:
https://github.com/tokend/web-client/pull/38#discussion_r253804830